docs/zope4/migration/zmi.rst

```diff
@@ -24,6 +24,13 @@ Example to use the info icon (i in a circle)::
 
     zmi_icon = 'fas fa-info-circle'
 
+If your product does not need to show a form when adding an object in the ZMI
+using the `Select type to add` dropdown, which only applies to a small
+percentage of available Zope products, you need to add another class variable
+to prevent the modal dialog from appearing::
+
+    zmi_modal = False
```

Comment:
As this attribute will be rarely used I'd like to have a name which tells ab bit more about its purpose. Maybe `zmi_show_modal_add_dialog` or `zmi_render_modal_dialog_on_add`.

Reply:
I have renamed the variable to `zmi_show_add_dialog`